rules/linux/env_binary_shell_evasion.toml

```
@@ -0,0 +1,25 @@
+[metadata]
+creation_date = "2022/02/24"
+maturity = "production"
+updated_date = "2022/02/24"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies Unix binary env absue to break out from restricted environments by spawning an interactive system shell"
```

```suggestion
description = "Identifies Linux binary env absue to break out from restricted environments by spawning an interactive system shell."
```

Can we also elaborate on these as to how this is dangerous or risky. Could also be worth mentioning that users rely on these to "live off the land" and to blend in with expected activity (as much as we love GTFObins, it may be less common in the security lexicon 🙂 )

---

```suggestion
description = "Identifies Linux binary env abuse to break out from restricted environments by spawning an interactive system shell"
```